.github/CONTRIBUTING.md

```diff
@@ -1,96 +1,66 @@
-
 You can find the pull request template [here](PULL_REQUEST_TEMPLATE.md).
 
-## Testing Your Build
-
-### node and npm Install
-
-1. [Install node](https://nodejs.org/en/download/), npm is bundled with the download
-2. Open a command prompt in your repo directory and run 'npm install'.
-
-### Environment Variables
-
-If your command requires environment variables please add them to the ```.env.example``` file for documentation.
-
-1. Copy the ```.env.example``` file to ```.env```
-2. You can do this manually or using the command line by typing ```cp .env.example .env```.
-3. Fill out the ```.env``` file with your environment variables.
-
-### Bot Application
-
-1. Log into the Discord website and navigate to the [applications page](https://discord.com/developers/applications)
-2. Click `New Application`
-3. Give your application a name
-4. Navigate to the `Bot` tab and click `Add Bot`. You will have to confirm by clicking `Yes, do it!`
-5. Click the `Copy` button underneath token. (Do not share this)
-6. Create a file called `.env` in your bot repo
-7. Inside the .env file, type `BOT_SECRET=TOKEN` replacing TOKEN with what you copied in step 6
-8. You may need to add the .env file to your gitignore if your IDE hasn't done it automatically.
-
-### Tokens
-
-Some commands may require additional tokens. If you would like to test them out on your bot, you must include the tokens inside your .env file. These commands include the metar, station and Wolfram Alpha commands. The steps below will explain how to set this up.
-
-#### AVWX (Metar and Station)
+## Setting up the Bot
+1. Log into the Discord website and navigate to the [applications page](https://discord.com/developers/applications).
+2. Click `New Application`.
+3. Give your application a name.
+4. Navigate to the `Bot` tab and click `Add Bot`. You will have to confirm by clicking `Yes, do it!`.
+5. Click the `Copy` button underneath token. (Do not share this).
+6. Create a file called `.env` in your bot repo.
+7. Inside the .env file, type `BOT_SECRET=TOKEN` replacing TOKEN with what you copied in `step 6.`
 
-1. Make a free account [here](https://avwx.rest/). Then, follow the steps [here](https://account.avwx.rest/getting-started) to get your token
-2. Inside the .env file, on a new line type `METAR_TOKEN=TOKEN` replacing TOKEN with what you copied in step 1
-3. Then, on another new line within the .env file, type `STATION_TOKEN=TOKEN` replacing TOKEN with what you copied in step 1
-
-#### Wolfram Alpha
-
-1. Select get API access [here](https://products.wolframalpha.com/api/) to create an account.
-2. Once you have an account you will need to get an AppID from [here](https://developer.wolframalpha.com/portal/myapps/)
-3. Inside the .env file, on a new line type `WOLFRAMALPHA_TOKEN=TOKEN` replacing TOKEN with your wolfram alpha AppID
+## Running the Bot
+2. Run `npm install` to install the dependencies.
+3. Run `npm run dev` to start the development build.
 
+## Database Setup
 ### MongoDB
```

Comment:
Any reason DB setup has moved above AVWX and WA?

Reply:
Felt it was more important than optional tokens. These require some commands to actually function instead of just 1 single command.

Reply:
Cool, that's fair!